Sming/SmingCore/Network/HttpRequest.cpp

```diff
@@ -136,6 +136,11 @@ HttpParseResult HttpRequest::parseHeader(HttpServer *server, pbuf* buf)
 		line = nextLine + 2;
 	} while(nextLine != -1);
 
+	if (getContentLength() > 0)
+	{
+		content = NetUtils::pbufStrCopy(buf, headerEnd + 4, getContentLength());
```

Comment:
Right... I think I need to handle that part when Header content is fragmented. But mostly for post data it comes in a single packet which I checked.